### PR TITLE
Release 8.5.0

### PR DESCRIPTION
**Pull request by wyardley**
#### Pull Request (PR) description
Release 8.5.0

### CODE REVIEW

**Comment by wyardley** [comment]
> Can we please include #761 before we do this release?

If it's breaking (meaning we have to go to 9.x), can we do it after, so that folks using older versions of Puppet can still get this bugfix?

File: lib/puppet/provider/rabbitmq_cli.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ def self.rabbitmqctl_list(resource, *opts)
   end
 
   # Retry the given code block 'count' retries or until the
-  # command suceeeds. Use 'step' delay between retries.
+  # command succeeeds. Use 'step' delay between retries.
```
Comment (ghoneycutt) [inline lib/puppet/provider/rabbitmq_cli.rb]: extra C is good, extra E is not :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v8.5.0](https://github.com/voxpupuli/puppet-rabbitmq/tree/v8.5.0) (2019-01-15)
```
Comment (ghoneycutt) [inline CHANGELOG.md]: Believe this should be updated by whomever does the release and not part of a pull request.

**Comment by ghoneycutt** [comment]
If I'm understanding @wyardley correctly, +1 to tagging a release (not a -rc version) before dropping Puppet 4 in a major version change.

File: lib/puppet/provider/rabbitmq_cli.rb (hunk 1 above)
Comment (wyardley) [reply lib/puppet/provider/rabbitmq_cli.rb]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (wyardley) [reply CHANGELOG.md]: Or to put it differently, I did the normal process, I think (https://voxpupuli.org/docs/#releasing-a-new-version-of-a-module)
Seems to track w/ the way the version looks in the changelog in https://github.com/voxpupuli/puppet-rabbitmq/pull/746/files

**Comment by travistie1** [comment]
Hi All, Is there any timeframe of when you will be deploying this 8.5.0 release to the Puppet Forge?
Just cause i need the fixes that you're releasing. I currently use Puppet 4.10.x and require the fixes for deploying & managing rabbitmq 3.7.9.

**Comment by travistie1** [comment]
> I need to rebuild the changelog again. I can’t do it until about Sunday, tho it’s fine if someone else from Vox has a chance.

That's fine as long as i can get a Release next week some time that would be great.

**Comment by wyardley** [comment]
Sorry for the delay, this is ready for review if someone has a sec.

**Comment by ahmet2mir** [comment]
It seems that puppet 6 nightly build failed.

```$ docker run --rm -it ubuntu:16.04 bash

root@6ae9bbeb2978:/# apt update
root@6ae9bbeb2978:/# apt install -y wget
root@6ae9bbeb2978:/# wget -O /tmp/puppet.deb http://nightlies.puppet.com/apt/puppet6-nightly-release-bionic.deb
root@6ae9bbeb2978:/# dpkg -i --force-all /tmp/puppet.deb
root@6ae9bbeb2978:/# apt update

root@6ae9bbeb2978:/# apt install puppet-agent
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Package puppet-agent is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'puppet-agent' has no installation candidate

```

**Comment by wyardley** [comment]
Yup, I can reproduce that locally and seems to affect other builds as well. I think it’s unrelated to our changes.

**Comment by korporationcl** [comment]
> > I need to rebuild the changelog again. I can’t do it until about Sunday, tho it’s fine if someone else from Vox has a chance.
> 
> That's fine as long as i can get a Release next week some time that would be great.

I'm actually waiting for this release too

**Comment by wyardley** [comment]
@bastelfreak do you want to approve these with the Puppet 6 nightly failures, or is there any progress on that issue?

**Comment by wyardley** [comment]
Thanks for everyone's patience. Some unrelated problems were causing those Travis failures.

https://forge.puppet.com/puppet/rabbitmq

**Pull request merged**